lib/core/src/display/scene/htmlobject.rs

```diff
@@ -0,0 +1,49 @@
+use super::Object;
+use crate::system::web::{create_element_as, Result};
```

Comment:
we're not using mutliple imports in a single line. This is because its much easier to manage one-import per line (disable, remove). When we will get rustfmt fixed some day it will automatically transform such imports to single-line ones.

Reply:
I am not sure I understood what you meant. Is the suggestion here to change it to

```
use crate::system::web::create_element_as;
use crate::system::web::Result;
```

?